rustls/src/webpki/client_verifier.rs

```diff
@@ -301,21 +290,19 @@ impl ClientCertVerifier for WebPkiClientVerifier {
         now: UnixTime,
     ) -> Result<ClientCertVerified, Error> {
         let cert = ParsedCertificate::try_from(end_entity)?;
-        let crls = borrow_crls(&self.crls);
 
-        let revocation = if crls.is_empty() {
+        let crl_refs = self.crls.iter().collect::<Vec<_>>();
+
+        let revocation = if self.crls.is_empty() {
             None
         } else {
-            let mut builder = webpki::RevocationOptionsBuilder::new(&crls)
-                .expect("invalid crls")
-                .with_depth(self.revocation_check_depth);
-            if matches!(
-                self.unknown_revocation_policy,
-                webpki::UnknownStatusPolicy::Allow
-            ) {
-                builder = builder.allow_unknown_status();
-            }
-            Some(builder.build())
+            Some(
+                webpki::RevocationOptionsBuilder::new(&crl_refs)
+                    .unwrap()
```

Comment:
Add a note about why this is okay?

Reply:
Good call. I think I had something like that in an earlier pass but lost it along the way. Fixed up.